# This simulates the interaction the robot would have with
# a guest when picking them up, such as:
# "Hello, guest! My name is servicebot, please follow me."

# Guest name
string guest_name

# Robot name
string robot_name

---
# Guest response
# This simulates the guest response, such as "Sure, I'll follow you"

# True if guest is following
bool success
